packages/core/bootstrap/src/lib/server.ts

```diff
@@ -61,11 +62,20 @@ export const initHandler =
       const metricsMeta = METRICS_ENABLED
         ? { metricsMeta: { requestOrigin: getClientIp(req) } }
         : {}
+      const ip = req.ip
```

Comment:
We should already be getting the request IP using `getClientIp(req)`, above. We can probably just get it once for both places